llama_stack/distribution/datatypes.py

```diff
@@ -117,6 +117,14 @@ class Provider(BaseModel):
     config: Dict[str, Any]
 
 
+class Logging(BaseModel):
```

Comment:
oh I meant just change the `key` not the classname. so `logging: Optional[LoggingConfig]`, sorry for being unclear and somewhat nitpicky.

Reply:
no worries, just changed it back